saleor/order/models.py

```diff
@@ -344,7 +344,7 @@ class OrderHistoryEntry(models.Model):
     order = models.ForeignKey(
         Order, related_name='history', on_delete=models.CASCADE)
     status = models.CharField(max_length=32, choices=OrderStatus.CHOICES)
-    comment = models.CharField(max_length=100, default='', blank=True)
+    content = models.TextField(default='', blank=True)
```

Comment:
We used to allow adding empty messages to track only the status changes in the past, but I think much better practice would be to require the message to make the entries meaningful. So `default=''` and `blank=True` could be removed. I would also drop the field `status` as I don't see much benefit from tracking it. We have only two order statuses now and they're simple to understand based on the delivery groups.